### PR TITLE
n-Queens problem in JavaScript

### PR DESCRIPTION
**Pull request by devRD**
N-Queens problem in JavaScript 

Issue #757

### CODE REVIEW

**Comment by devRD** [comment]
I included the issue number, editing the PR description

File: Queens_Problem_js/Queens_Problem.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
+/* N-Queen's Problem
```
Comment (jainaman224) [inline Queens_Problem_js/Queens_Problem.js]: Please fix folder name.

File: Queens_Problem_js/Queens_Problem.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+* Language: JavaScript
+*/
+
+/*To check if the queen is under attack, sideways or diagonally*/
+const underAttack = (board, row, col) => {
+
```
Comment (jainaman224) [inline Queens_Problem_js/Queens_Problem.js]: Please use 4 space indentation throughout the file.

File: Queens_Problem_js/Queens_Problem.js (hunk 1 above)
Comment (devRD) [reply Queens_Problem_js/Queens_Problem.js]: what will be the folder name?

**Comment by jainaman224** [comment]
@devRD Which editor do you use?

**Comment by jainaman224** [comment]
@devRD Indentation is still not fixed.

**Comment by devRD** [comment]
I tried auto-indenting the file with 4 space indentation. I'm not sure how to fix it

**Comment by jainaman224** [comment]
Can you connect with us on slack?

**Comment by devRD** [comment]
yes

**Comment by jainaman224** [comment]
You are almost there. Just some comments are indented incorrectly.

**Comment by devRD** [comment]
Is it alright now?

File: Queens_Problem_js/Queens_Problem.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    //Checks if all queens are placed
+    if(col >= board.length)
+        return true;
+
+    for(let i = 0; i < board.length; i++){
+        if(!underAttack(board, i, col)){
+
+        //Placing the queens in the correct position
+        board[i][col] = 'Q';
```
Comment (jainaman224) [inline Queens_Problem_js/Queens_Problem.js]: Indentation got at wrong from here.

**Comment by devRD** [comment]
@jainaman224  Can you check now?

**Pull request merged**